.env.example

```diff
@@ -13,3 +13,10 @@ ANALYTICS_MODE=
 # Use `trace` to get verbose logging or `info` to show less
 LOG_LEVEL=debug
 LOGDNA_INGESTION_KEY=
+OPENAI_API_HOST=https://api.openai.com
+OPENAI_API_KEY=
+CHATGPT_USER_PROMPT_FOR_IMPORTANT_WORDS="I need your help to find important words (e.g. unique adjectives) from github issue below and I want to parse them easily so please separate them using #(No other contexts needed). Please separate the words by # so I can parse them easily. Please answer simply as I only need the important words. Here is the issue content.\n"
+CHATGPT_USER_PROMPT_FOR_MEASURE_SIMILARITY='I have two github issues and I need to measure the possibility of the 2 issues are the same content (No other contents needed and give me only the number in %).\n Give me in number format and add % after the number.\nDo not tell other things since I only need the number (e.g. 85%). Here are two issues:\n 1. "%first%"\n2. "%second%"'
+SIMILARITY_THRESHOLD=80
```

Comment:
- To make it clear, it should be stored in `ubiquibot-config.json` because our partners should be able to configure these to accommodate per repository. This is because certain partners might be able to tweak the config to increase similarity accuracy. https://github.com/ubiquity/ubiquibot/pull/608/files#r1292178783
- If it is in `.env` that implies that we must change it in our production instance Netlify environment config, which affects every partner.